arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -103,5 +114,45 @@ public async Task PostCollectionAsync_ShouldThrow_WhenCollectionNameInvalid()
             });
             Assert.Equal(1208, ex.ApiError.ErrorNum);
         }
+
+        [Fact]
+        public async Task TruncateCollectionAsync_ShouldSucceed()
+        {
+            // add a document
+            var response = await _adb.Document.PostDocumentAsync<object>(
+                _testCollection,
+                new { test = 123 });
+            Assert.NotNull(response._id);
+
```

Comment:
Should we also count here to assert that there is one document in the collection?

Reply:
I don't think we need to do that